### ISSUE TITLE
Right Click Menu blocked by Stage

### ISSUE THREAD

**Issue by tonyzhigit** [ScratchAddons/ScratchAddons#5484]
### Description

I can't see the whole menu because some buttons are blocked by the stage.

### Steps to reproduce

1. Open the editor.
2. Create folders.
3. Right-click on any sprite.

### Demonstration(s) and additional context

![image](https://user-images.githubusercontent.com/100283902/209425141-b4361da0-97da-4cbc-9157-a1611f430caa.png)

### Environment

Browser: Chome 110.0.5478.4 dev x64 (cohort: Dev) 
Scratch Addons version: v1.29.0
Operating system: Windows 11 21H2

**Comment by mxmou** [comment]
Duplicate of #5185
